src/interpreter_funcs/voxel_metaballs.rs

```diff
@@ -142,10 +144,10 @@ impl Func for FuncVoxelMetaballs {
                 name: "Volume range",
                 description: "Materializes voxels with value within the given range.",
                 refinement: ParamRefinement::Float2(Float2ParamRefinement {
-                    min_value: Some(0.0),
-                    max_value: Some(100.0),
-                    default_value_x: Some(0.5),
-                    default_value_y: Some(2.0),
+                    min_value: None,
+                    max_value: None,
+                    default_value_x: Some(f32::INFINITY),
+                    default_value_y: Some(f32::INFINITY),
```

Comment:
This break the UI. If you try to change this, the value becomes NaN and can't be set back to infinity.